.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+<!--
+
+A crisp summary
+
+Fixes #
+Ref #
+-->
+
+<!--
+Apply labels!
+-->
+
+## Checklist
+
+- [ ] Labels applied correctly? See the [contributing guidlines](../CONTRIBUTING.md).
+- [ ] Workspace compiles `cargo build --workspace`?
+- [ ] Workspace tests `cargo test --workspace`?
+- [ ] Applied `cargo +nightly fmt` with the version from CI?
+- [ ] Ran `cargo spellcheck` without any fallout? If any, update `.spellcheck/lingua.dic`.
+- [ ] Fixed any fallout caused by these changes in `cumulus`?
+- [ ] Referenced relevant companion pullrequests of `substrate` and `cumulus`?
+- [ ] Does the change mandate an _architecture design record_ being committed?
```

Comment:
A link to a definition of _architecture design record_ and how to know when a change is mandatory?

Reply:
Yep, currently writing that up and creating a template for that too :)

Reply:
Done https://github.com/paritytech/polkadot/pull/5390/commits/0d5b720886df235b28272bf962b50ea221ae15ad